scripts/make-env

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/bash
```

Comment:
Why did you have to turn off auto-bail? I think it's really useful switch to have always on as you can be certain things you do have exited properly and that any expected failures are handled gracefully.